From 2592401c5ab776090dabc2df7f705cc301552bcd Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Thu, 15 Dec 2022 14:58:02 +0100
Subject: [PATCH] fdts: stm32mp157c-bytedevkit*: Add symbolic links

meta-st-stm32mp assumes that the device tree file of the machine has the
same name for all tf-a, u-boot, and kernel. Since we don't have a separate
board and device trees, create links to make it work.

Signed-off-by: Daniel Ammann <daniel.ammann@bytesatwork.ch>
---
 fdts/stm32mp157c-bytedevkit-v1-1-fw-config.dts | 1 +
 fdts/stm32mp157c-bytedevkit-v1-1.dts           | 1 +
 fdts/stm32mp157c-bytedevkit-v1-3-fw-config.dts | 1 +
 fdts/stm32mp157c-bytedevkit-v1-3.dts           | 1 +
 4 files changed, 4 insertions(+)
 create mode 120000 fdts/stm32mp157c-bytedevkit-v1-1-fw-config.dts
 create mode 120000 fdts/stm32mp157c-bytedevkit-v1-1.dts
 create mode 120000 fdts/stm32mp157c-bytedevkit-v1-3-fw-config.dts
 create mode 120000 fdts/stm32mp157c-bytedevkit-v1-3.dts

diff --git a/fdts/stm32mp157c-bytedevkit-v1-1-fw-config.dts b/fdts/stm32mp157c-bytedevkit-v1-1-fw-config.dts
new file mode 120000
index 000000000..5bcb26d07
--- /dev/null
+++ b/fdts/stm32mp157c-bytedevkit-v1-1-fw-config.dts
@@ -0,0 +1 @@
+stm32mp157c-bytedevkit-fw-config.dts
\ No newline at end of file
diff --git a/fdts/stm32mp157c-bytedevkit-v1-1.dts b/fdts/stm32mp157c-bytedevkit-v1-1.dts
new file mode 120000
index 000000000..871e8ef8f
--- /dev/null
+++ b/fdts/stm32mp157c-bytedevkit-v1-1.dts
@@ -0,0 +1 @@
+stm32mp157c-bytedevkit.dts
\ No newline at end of file
diff --git a/fdts/stm32mp157c-bytedevkit-v1-3-fw-config.dts b/fdts/stm32mp157c-bytedevkit-v1-3-fw-config.dts
new file mode 120000
index 000000000..5bcb26d07
--- /dev/null
+++ b/fdts/stm32mp157c-bytedevkit-v1-3-fw-config.dts
@@ -0,0 +1 @@
+stm32mp157c-bytedevkit-fw-config.dts
\ No newline at end of file
diff --git a/fdts/stm32mp157c-bytedevkit-v1-3.dts b/fdts/stm32mp157c-bytedevkit-v1-3.dts
new file mode 120000
index 000000000..871e8ef8f
--- /dev/null
+++ b/fdts/stm32mp157c-bytedevkit-v1-3.dts
@@ -0,0 +1 @@
+stm32mp157c-bytedevkit.dts
\ No newline at end of file
-- 
2.39.2

